src/components/SellerCard/SellerCard.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react"
+import "./SellerCard.css"
+
+export default function SellerCard(props) {
+    // unit enum 
+    const UNIT_TYPE = Object.freeze({
+        VOLUME: 1,
+        WEIGHT: 0,
+        UNKNOWN: -1
+    })
```

Comment:
I've see this duplicated a few places, try to find a way to share this

Reply:
I already have the enum in my conversion utility file so I'll just export that enum. I think importing the enum will be a lot more concise than passing it in as props